charts/opentelemetry-operator/README.md

```diff
@@ -18,34 +18,7 @@ certificate that the API server is configured to trust. There are three ways for
     In this way, cert-manager will generate a self-signed certificate. _See [cert-manager installation](https://cert-manager.io/docs/installation/kubernetes/) for more details._
   - You can also provide your own Issuer by configuring the `admissionWebhooks.certManager.issuerRef` value. You will need
     to specify the `kind` (Issuer or ClusterIssuer) and the `name`. Note that this method also requires the installation of cert-manager.
-  - The last way is to manually modify the secret where the TLS certificate is stored. Make sure you set `admissionWebhooks.certManager.enabled` to `false` first.
-    - Create the namespace for the OpenTelemetry Operator and the secret
-      ```console
-      $ kubectl create namespace opentelemetry-operator-system
-      ```
-    - Config the TLS certificate using `kubectl create` command
-      ```console
-      $ kubectl create secret tls opentelemetry-operator-controller-manager-service-cert \
-          --cert=path/to/cert/file \
-          --key=path/to/key/file \
-          -n opentelemetry-operator-system
-      ```
-      You can also do this by applying the secret configuration.
-      ```console
-      $ kubectl apply -f - <<EOF
-      apiVersion: v1
-      kind: Secret
-      metadata:
-        name: opentelemetry-operator-controller-manager-service-cert
-        namespace: opentelemetry-operator-system
-      type: kubernetes.io/tls
-      data:
-        tls.crt: |
-            # your signed cert
-        tls.key: |
-            # your private key
-      EOF
-      ```
+  - The last way is to use self-signed certificate by configuring the `admissionWebhooks.certManager.enabled` to `false`.
```

Comment:
@Allex1 @JaredTan95 it feels kinda strange for the chart to create a new object when a existing setting is set to false. Is there a way to implement this change so that the secret can be created based on a new entry in the values.yaml?

Reply:
@TylerHelmuth this can probably be achieved but to me it makes sense the way it is defined in the template : admissionWebhooks -> enabled and certManager -> disabled.
Maybe we can make it more clear in the docs.

Reply:
@TylerHelmuth a secret was manually created before as well with the exact same name `opentelemetry-operator-controller-manager-service-cert` , so an upgrade would just update that secret. We could explain in values.yaml via comments what the `admissionWebhooks.certManager.enabled: false` does in order to avoid confusion

As a last resort we could add another flag that the template would validate like `admissionWebhooks.selfSignedCert.create: True` to be mutually exclusive to `admissionWebhooks.certManager.enabled: False` and update the docs if you really think it's useful .

Reply:
If `admissionWebhooks.certManager.enabled: False` always implies `admissionWebhooks.selfSignedCert.create: True` then `admissionWebhooks.selfSignedCert.create` probably isn't useful, but I do think we should make it clear via the docs and values.yaml that `admissionWebhooks.certManager.enabled: False` means the secret will be created for you.

Reply:
@JaredTan95 can you update the documentation? Once that's good we'll be good to go.

Reply:
okay